src/main/java/io/github/cottonmc/energy/api/ElectricalEnergyType.java

```diff
@@ -1,20 +1,35 @@
 package io.github.cottonmc.energy.api;
 
+import net.minecraft.text.TextComponent;
+import net.minecraft.text.TranslatableTextComponent;
+
 public class ElectricalEnergyType implements EnergyType {
 
-    private final int size;
+    private final int lvPerPacket;
 
-    public ElectricalEnergyType(int maxSize) {
-        this.size = maxSize;
+    public ElectricalEnergyType(int lvPerPacket) {
+        this.lvPerPacket = lvPerPacket;
     }
 
     public int getPacketSize() {
-        return size;
+        return 1;
     }
 
-
     @Override
     public int getMaximumPacketSize() {
-        return this.size;
+        return 1;
+    }
+
+    /**
+     * @return the amount of energy in this packet if it were converted to LV
+     */
+    public int getLVAmount() {
+        return lvPerPacket;
+    }
+
+    @Override
+    public TextComponent getDisplayAmount(int amount) {
+        //TODO: handle KWU, MWU, and GWU
+        return new TranslatableTextComponent("info.cotton.energy.electrical.amount", lvPerPacket * amount);
```

Comment:
Why are you multiplying amount with lvPerPacket? Amount will be the absolute value and doesn't have anything to do with this types max packet size.

Reply:
lvPerPacket is a measure of how many work units are contained per type of that packet. Falk refactored it so each electrical energy type cares a fixed amount of power.